runtime/oti/jclprots.h

```diff
@@ -711,7 +711,7 @@ UDATA initializeRequiredClasses(J9VMThread *vmThread, char* libName);
 /* J9SourceJclDefineClass*/
 extern J9_CFUNC jclass 
 defineClassCommon (JNIEnv *env, jobject classLoaderObject,
-	jstring className, jbyteArray classRep, jint offset, jint length, jobject protectionDomain, UDATA options, J9Class *hostClass, J9ClassPatchMap *patchMap);
+	jstring className, jbyteArray classRep, jint offset, jint length, jobject protectionDomain, UDATA * options, J9Class *hostClass, J9ClassPatchMap *patchMap, UDATA allowedBitsForClassName);
```

Comment:
For consistency, please remove the space after the `*` in options (here and in the function declaration).

Reply:
Fixed as suggested above.